app/services/users.js

```diff
@@ -69,3 +69,43 @@ exports.changeRole = (role, email) =>
     logger.error('Error trying to update the user');
     throw errors.databaseError('Error processing request in database.');
   });
+
+exports.findUser = email =>
+  User.findOne({
+    where: { email },
+    raw: true,
+    attributes: ['id', 'firstName', 'lastName', 'email', 'password', 'role']
+  }).catch(err => {
+    logger.info('Error trying to find the user');
+    throw errors.databaseError(`${err}`);
+  });
+
+exports.findAllUser = (limit, offset) =>
+  User.findAndCountAll({
+    limit,
+    offset,
+    raw: true,
+    attributes: ['id', 'firstName', 'lastName', 'email']
+  }).catch(err => {
+    logger.info('Error trying to find the user');
+    throw errors.databaseError(`${err}`);
+  });
+
+exports.AdminUserRegister = user =>
+  User.create({ ...user, role: 'admin' }).catch(() => {
```

Comment:
why another method for admin user?
if you have 10 roles you would create 10 methods?
try to reuse the original create user

Reply:
It was my mistake, I was using it but in the end reuse userRegister.